src/clib/old_tests/job_queue/test_job_torque.cpp

```diff
@@ -7,6 +7,7 @@
 
 void test_option(torque_driver_type *driver, const char *option,
                  const char *value) {
+    printf("test_option %s %s\n", option, value);
```

Comment:
I could not understand why this printf is kept...was it not meant?

Reply:
thanks, it should not be there :)